### PR TITLE
ansible: move ansible to be pypi-based, with selectable runtime

### PR DESCRIPTION
**Pull request by adfernandes**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [x] security fix

###### Tested on

macOS 10.12

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/ansible_select/Portfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           select 1.0
+
+name                ansible_select
+version             0.1
+categories          sysutils
```
Comment (blair) [inline python/ansible_select/Portfile]: There's an error in the build, try changing to categories-append.

File: python/ansible_select/Portfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+platforms           darwin
+supported_archs     noarch
+license             BSD
+maintainers         {adfernandes @adfernandes} openmaintainer
```
Comment (blair) [inline python/ansible_select/Portfile]: Keep this consistent with py-ansible?

File: python/py-ansible/Portfile [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    fs-traverse f ${worksrcpath} {
+        if {[file isfile ${f}] && [regexp {\.(py|yml|yaml)$} ${f} match]} {
+            reinplace -q "s#/etc/ansible#${prefix}/etc/ansible#g" ${f}
+            reinplace -q "s#/usr/share/ansible#${prefix}/share/ansible#g" ${f}
+        }
+    }
+}
+
+if {${name} ne ${subport}} {
+
```
Comment (blair) [inline python/py-ansible/Portfile]: Remove empty lines.

File: python/ansible_select/Portfile (hunk 1 above)
Comment (adfernandes) [reply python/ansible_select/Portfile]: Changed...

... but the Travis failure looked like it was a general timeout. Not sure if it's linked.

We'll see when the updated PR build finishes, below, I guess...!

**Comment by adfernandes** [comment]
Hey, @blair - I'm looking at the output of [the Travis job, above](https://travis-ci.org/macports/macports-ports/builds/310315522?utm_source=github_status&utm_medium=notification), and it appears to pass except for the following:

```
/Users/travis/.travis/job_stages: line 57: shell_session_update: command not found
```

That doesn't appear (to me) to be a problem with the port changes, more a problem with Travis, no?

I'd like to get this port committed, but am not sure how to proceed from here.

**Pull request merged**